dashboard/views.py

```diff
@@ -125,18 +146,19 @@ def get_course_info(request, course_id=0):
     if total_weeks > settings.MAX_DEFAULT_WEEKS:
         logger.debug(f'{total_weeks} is greater than {settings.MAX_DEFAULT_WEEKS} setting total weeks to default.')
         total_weeks = settings.MAX_DEFAULT_WEEKS
-
     resp['current_week_number'] = current_week_number
     resp['total_weeks'] = total_weeks
-    resp['course_view_options'] = CourseViewOption.objects.get(course=course).json(include_id=False)
+    resp['course_view_options'] = get_course_view_options(request.user.is_staff, course)
     resp['resource_types'] = course_resource_list
-
     course_users_list = User.objects.filter(course_id=course_id)
     resp['course_user_exist'] = 1 if len(course_users_list) > 0 else 0
 
     return HttpResponse(json.dumps(resp, default=str))
 
 
+
+
+
```

Comment:
Peer review has taught me that I'm supposed complain about blank lines :)

Reply:
Done